plugin/PDF/PdfDownload.php

```diff
@@ -42,21 +43,28 @@ class PdfDownload
     /**
      * @var HTTPHeaders
      */
-    private $_httpHeadersGeneric;
+    private $_pHttpHeadersGeneric;
+    /**
+     * @var WPOptionWrapperDefault
+     */
+    private $_pWPOptionWrapper;
 
     /**
      * @param PdfDocumentFetcher $pPdfDocumentFetcher
      * @param PdfDocumentModelValidator $pPdfDocumentModelValidator
-     * @param HTTPHeaders $httpHeadersGeneric
+     * @param HTTPHeaders $pHttpHeadersGeneric
+     * @param WPOptionWrapperDefault $pWPOptionWrapper
      */
 	public function __construct(
 		PdfDocumentFetcher $pPdfDocumentFetcher,
 		PdfDocumentModelValidator $pPdfDocumentModelValidator,
-        HTTPHeaders $httpHeadersGeneric)
+        HTTPHeaders $pHttpHeadersGeneric,
+        WPOptionWrapperDefault $pWPOptionWrapper = null)
```

Comment:
`WPOptionWrapperBase` should be requested here and ` = null` should be removed.

Reply:
I removed it

Reply:
`WPOptionWrapperDefault $pWPOptionWrapper = null` should be `WPOptionWrapperBase $pWPOptionWrapper`

Reply:
I updated it, thanks